docs/classifieds/index.md

```diff
@@ -2,14 +2,18 @@
 
 > Classifieds or Classified advertising is free of charge in the newspaper. Keep them short and to the point.
 
+[<h2>**AraCon is Coming in January**</h2>](https://aracon.one)
+[REGISTER BUILDING ORGANIZATIONS & GOVERNANCE OF THE FUTURE](https://aracon.one/registration/) 29th-30th of January 2019 in Berlin Berlin, Germany Jan 29-30.2019 The Aragon Conference will bring together people who embody and wish to shape the Aragon community. Those who are building and supporting the pseudonymous, secure, trustless future — the decentralized web.
+[Learn More](https://aracon.one/faq) |
+
 [<h2>**Senior software dev/architect at P2P Models**</h2>](https://p2pmodels.eu/jobs-senior-dev/) |
 :-----------|
 [P2P Models](https://p2pmodels.eu/) is a 5-year (2018-2022) EU-funded (1.5M€) project around blockchain and the commons, which will be based at the Universidad Complutense de Madrid (Spain). More in this [thread](https://twitter.com/p2pmod/status/1017802485422067713). **We are looking for a Senior Software Developer/Architect** to lead the development of a blockchain-based framework to build collaborative Decentralized Autonomous Organizations. This role involves to establish the architectural foundations and develop a software framework to build modular components of DAOs, with a focus on supporting collaboration and communities. |
 [Learn More](https://p2pmodels.eu/jobs-senior-dev/) |
 
 [<h2>**Project Manager / Communication Manager at P2P Models**</h2>](https://p2pmodels.eu/jobs-project-manager/) |
```

Comment:
Should be removed, was included in the previous issue and hasn't been re-submitted for this Issue.

Reply:
You made the changes to the AraCon listing, do you mean the P2P model listing should be removed?